src/index.ts

```diff
@@ -219,9 +223,18 @@ export const build = <FactoryResultType>(
 
     return buildTimeMapFunc(postBuild(afterTraitPostBuildFields));
   };
+
   builder.reset = () => {
     sequenceCounter = 0;
   };
+
+  builder.many = (
+    times: number,
+    buildTimeConfig: BuildTimeConfig<FactoryResultType>
+  ): FactoryResultType[] => {
+    return new Array(times).fill(builder(buildTimeConfig));
```

Comment:
If I'm not wrong, this will create one instance and fill the array with that same object. So `userBuilder.many(20)` will return an array of 20 times the same exact user.

Probably this should fix it:

```suggestion
    return new Array(times).fill(0).map(() => builder(buildTimeConfig));
```